packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -68,18 +69,23 @@ export class NextCookies extends Cookies {
     super(response.headers.get('cookie'))
     this.response = response
   }
-  get = (...args: Parameters<Cookies['get']>) => {
+  get(...args: Parameters<Cookies['get']>) {
     return this.getWithOptions(...args).value
   }
-  getWithOptions = (
-    ...args: Parameters<Cookies['get']>
-  ): GetWithOptionsOutput => {
+  getAll(): GetWithOptionsOutput[] {
+    const all: GetWithOptionsOutput[] = []
+    for (const key of this.keys()) {
```

Comment:
maybe something like this?

const all: GetWithOptionsOutput[] = this.keys().map(key => this.getWithOptions(key))
return all

Reply:
`this.keys()` gives a `MapIterator`, not an array, so the `.map` method does not exist. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/keys